packages/components/src/styles/components/form/super-select.scss

```diff
@@ -0,0 +1,490 @@
+      background-image: var(--token-form-select-background-image-data-url-disabled);
+      border-color: var(--token-form-control-disabled-border-color);
+      box-shadow: none;
+      cursor: not-allowed;
+      opacity: 1;
+
+      .ember-power-select-selected-item {
+        opacity: inherit;
+      }
+    }
+  }
+
+  // DROPDOWN
+  .ember-basic-dropdown-content {
+    max-width: var(--hds-form-super-select-dropdown-max-width);
+    margin: 0;
+    border: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
+    border-radius: var(--token-form-control-border-radius);
+
+    &.ember-basic-dropdown-content--in-place {
+      box-shadow: var(--token-elevation-high-box-shadow), var(--token-elevation-high-box-shadow);
+    }
+
+    &.ember-basic-dropdown-content--below {
+      margin-top: 4px;
+    }
+
+    &.ember-basic-dropdown-content--above {
+      top: unset !important; // override inline style (TODO: update in JS)
+      bottom: 100%;
+      margin-bottom: 4px;
+    }
+
+    // hide first group separator line
+    > .ember-power-select-options > .hds-form-super-select__option-group:first-child::before {
+      display: none;
+    }
+  }
+
+  // Options List
+  .ember-power-select-options {
+    padding: 3px 0;
+
+    .ember-power-select-options {
+      padding: 0;
+    }
+  }
+
+  // Options Item
+  .ember-power-select-option {
+    position: relative;
+    min-height: $hds-super-select-item-height;
+    padding: 7px 7px 7px 39px; // top, right, bottom, left
+    line-height: var(--token-typography-body-200-line-height);
+    overflow-wrap: break-word; // when used with flex parent, content may overflow (therefore not using flex layout on parent)
+    border: none; // guard against border in default theme
+
+    + .ember-power-select-option {
+      margin-top: 8px;
+    }
+
+    // states
+    &[aria-current="true"] {
+      color: var(--token-form-control-base-foreground-value-color);
+      background-color: var(--token-form-control-base-surface-color-default);
+    }
+
+    &[aria-selected="true"] {
+      background-color: var(--token-form-control-base-surface-color-default);
+    }
+
+    &[aria-current="true"],
+    &:hover {
+      border: none; // guard against border in default theme
+
+      // left border
+      &::after {
+        position: absolute;
+        top: 6px;
+        bottom: 6px;
+        left: 3px;
+        width: 2px;
+        background-color: var(--token-color-foreground-action-hover);
+        border-radius: 500px; // Makes it look slightly rounded on the ends (doesn't match Figma exactly)
+        content: "";
+      }
+    }
+
+    &:active {
+      // left border
+      &::after{
+        background-color: var(--token-color-foreground-action-active);
+      }
+    }
+  }
+
+  // Option Groups (similar to HTML optgroup)
+
+  .hds-form-super-select__option-group {
+    &:first-child .hds-form-super-select__option-group-title {
```

Comment:
In the current shape I don't see any `margin-top` resets around L115 – some changes may have been introduced since them, but if you remember what was the redundancy about I can investigate further

Reply:
This seems to be outdated so I will close for now.